src/components/DeleteHelicopter.jsx

```diff
@@ -0,0 +1,87 @@
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import {
+  fetchHelicopters,
+  deleteHelicopter,
+  updateHelicopters,
+} from "../features/helicopters/helicopterSlice";
+import LayoutComponent from "../Layout";
+import { Button, Table, Popconfirm } from 'antd';
+import { QuestionCircleOutlined } from "@ant-design/icons";
+
+const DeleteHelicopterComponent = () => {
```

Comment:
- [ ] OPTIONAL: The session cookies feature has some issues.

In order to login again, I need to try multiple times, and I need experiment deleting cookies, local storage, etc until I can finally log in to a previously created user. I recommend taking a look at this before going to final evaluation.